packages/sources/elwood/src/transport/lwba.ts

```diff
@@ -0,0 +1,165 @@
+import { BaseEndpointTypesLwba } from '../endpoint/crypto-lwba'
+import { WebSocketTransport } from '@chainlink/external-adapter-framework/transports'
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { EndpointContext } from '@chainlink/external-adapter-framework/adapter'
+import axios from 'axios'
+
+const logger = makeLogger('ElwoodWsLwba')
+
+const DEFAULT_TRANSPORT_NAME = 'default_single_transport'
+
+export type SubscribeRequest = {
```

Comment:
It seems that this and bellow types are identical with `crypto` transport as well. Maybe we can move them to a shared `utils.ts` in transport folder ?